packages/dataviews/src/dataviews-layouts/table/column-header-menu.tsx

```diff
@@ -98,12 +99,27 @@ const _HeaderMenu = forwardRef( function HeaderMenu< Item >(
 		<Menu>
 			<Menu.TriggerButton
 				render={
-					<Button
-						size="compact"
-						className="dataviews-view-table-header-button"
-						ref={ ref }
-						variant="tertiary"
-					/>
+					field.description ? (
+						( props: any ) => (
+							<Tooltip text={ field.description } placement="top">
+								<Button
+									size="compact"
+									className="dataviews-view-table-header-button"
+									ref={ ref }
+									variant="tertiary"
+									description={ field.description }
+									{ ...props }
+								/>
+							</Tooltip>
+						)
```

Comment:
```suggestion
						( { children }: { children: ReactNode } ) => (
							<Tooltip text={ field.description } placement="top">
								<Button
									size="compact"
									className="dataviews-view-table-header-button"
									ref={ ref }
									variant="tertiary"
									description={ field.description }
								>
									{ children }
								</Button>
							</Tooltip>
						)
```

Nit:

Reply:
Thanks, but it does not seem to work. The header becomes unclickable because we don't pass relevant props (with this, we only pass `children`)